src/EditorFeatures/CSharpTest/Structure/RegionDirectiveStructureTests.cs

```diff
@@ -75,8 +75,8 @@ class Classic
         }
 
         [WorkItem(953668, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/953668")]
-        [Fact, Trait(Traits.Feature, Traits.Features.Outlining)]
-        public async Task RegionsShouldBeCollapsedByDefault()
+        [Theory, CombinatorialData, Trait(Traits.Feature, Traits.Features.Outlining)]
+        public async Task RegionsShouldBeCollapsedByDefault(bool CollapseRegionsWhenFirstOpened)
```

Comment:
I'm really surprised this didn't cause a build error

```suggestion
        public async Task RegionsShouldBeCollapsedByDefault(bool collapseRegionsWhenFirstOpened)
```